### PR TITLE
Readme: Update copyright year

### PR DESCRIPTION
**Pull request by MarcelBochtler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -791,7 +791,7 @@ to any of your questions we recommend you
 
 # License
 
-Copyright (C) 2017-2022 [The ORT Project Authors](./NOTICE).
+Copyright (C) 2017-2023 [The ORT Project Authors](./NOTICE).
```
Comment (sschuberth) [inline README.md:794]: Also see my comment at https://github.com/oss-review-toolkit/ort-config/pull/93#discussion_r1062310339. So should we simply drop the end year instead?

Comment (mnonnenmacher) [reply README.md:794]: I would be fine with keeping the end year in the README, not for legal reasons but to emphasize that the project is active.

**Review by sschuberth — APPROVED** [review]

**Comment by mnonnenmacher** [comment]
@sschuberth The Markdown link check is failing with:
```
ERROR: 2 dead links found!
[✖] https://fossid.com/ → Status: 0
[✖] https://fossid.com/open-source-compliance/ → Status: 0
```

I also saw this the last time I was editing the README, do you remember if we had this issue before?

**Comment by sschuberth** [comment]
I also saw this bogus failure in one of my recent PRs, but not before that.

**Comment by MarcelBochtler** [comment]
Merging despite the bogus markdown link tests.

**Pull request merged**